tests/services/inventory/crawling_test.py

```diff
@@ -490,7 +490,6 @@ def test_crawl_cai_api_polling_disabled(self):
             'compute_router': {'resource': 1},
             'compute_sslcertificate': {'resource': 1},
             'compute_targethttpproxy': {'resource': 1},
-            'compute_targethttpsproxy': {'resource': 1},
```

Comment:
This should not be removed

The forwardingrule entry below should be increased to 2, that's where the new global forwarding rule shows up.

You also need to add forwardingrule with a resource count of 2 to the test_cai_crawl_to_memory test.

Reply:
Done